cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -222,6 +229,150 @@ def _add_conditional_storage_widgets(
+                metric_name="IamPolicyErrors",
+                filter_pattern=_generate_metric_filter_pattern(launch_failure_event_type, "iam-policy-errors"),
+                metric_value=metric_value,
+            ),
+            _CustomMetricFilter(
+                metric_name="AmiLargerThanRootVolume",
+                filter_pattern=_generate_metric_filter_pattern(launch_failure_event_type, "custom-ami-errors"),
+                metric_value=metric_value,
+            ),
+            _CustomMetricFilter(
+                metric_name="VcpuLimit",
+                filter_pattern=_generate_metric_filter_pattern(launch_failure_event_type, "vcpu-limit-failures"),
+                metric_value=metric_value,
+            ),
+            _CustomMetricFilter(
+                metric_name="VolumeLimit",
+                filter_pattern=_generate_metric_filter_pattern(launch_failure_event_type, "volume-limit-failures"),
+                metric_value=metric_value,
+            ),
+            _CustomMetricFilter(
+                metric_name="NodeCapacityInsufficient",
+                filter_pattern=_generate_metric_filter_pattern(launch_failure_event_type, "ice-failures"),
+                metric_value=metric_value,
+            ),
+            _CustomMetricFilter(
+                metric_name="OtherLaunchedInstanceFailures",
+                filter_pattern=_generate_metric_filter_pattern(launch_failure_event_type, "other-failures"),
+                metric_value=metric_value,
+            ),
+        ]
+
+        # custom_script_errors = [  # TODO: Change Me
+        #     _CustomMetricFilter(
+        #         metric_name="CannotRetrieveCustomScript",
+        #         filter_pattern=custom_scripts_failure_event_pattern,
+        #         metric_value="1",
+        #     ),
+        #     _CustomMetricFilter(
+        #         metric_name="ErrorWithCustomScript",
+        #         filter_pattern=custom_scripts_failure_event_pattern,
+        #         metric_value="1",
+        #     ),
+        # ]
+
+        compute_node_events = [
+            _CustomMetricFilter(
+                metric_name="ReplacementTimeoutExpires",
+                filter_pattern=_generate_metric_filter_pattern(
+                    "protected-mode-error-count", "static-replacement-timeout-error"
+                ),
+                metric_value=metric_value,
+            ),
+            _CustomMetricFilter(
+                metric_name="ResumeTimeoutExpires",
+                filter_pattern=_generate_metric_filter_pattern(
+                    "protected-mode-error-count", "dynamic-resume-timeout-error"
+                ),
+                metric_value=metric_value,
+            ),
+            _CustomMetricFilter(
+                metric_name="EC2MaintenanceEvent",
+                filter_pattern=_generate_metric_filter_pattern(
+                    "nodes-failing-health-check-count", "ec2_health_check", failure_name="health-check-type"
+                ),
```

Comment:
Would it be easier for you if I changed the failure_name to be consistent with the node_launch_failure_count?